src/manager/app/models/volume.py

```diff
@@ -0,0 +1,51 @@
+import os
+from typing import Optional
+
+from pydantic import BaseModel, computed_field
+
+STATE_HOST_DIR = os.getenv("STATE_HOST_DIR")
+
+
+class RAGAppVolumeConfig(BaseModel):
+    name: str
+    container_data_path: str = "/app/data"
+    container_config_path: str = "/app/config"
+    container_storage_path: str = "/app/storage"
+    container_output_path: str = "/app/output"
+    host_mount_prefix: Optional[str] = STATE_HOST_DIR
```

Comment:
how about keeping the same name to avoid confusion?
```suggestion
    state_host_dir: Optional[str] = STATE_HOST_DIR
```